src/less/lite-ui/buttons.less

```diff
@@ -17,10 +18,10 @@
   &:hover,
   &:focus,
   &.focus {
-    color: @btn-default-color;
+    color: #333;
```

Comment:
Why did you change variables to constants?

Reply:
oh, ooopss, mostly likely copy and pasted from the css into the less files. Would you like them changed back?

Reply:
It would be better if you could revert them. Thanks.

Reply:
No worries, I'll get onto it.